### PR TITLE
RET-2618

### PR DESCRIPTION
**Pull request by ikirsanov**
###Jira Link
https://tools.hmcts.net/jira/browse/RET-2618

### Change description
Allow the Claimant to check the answers they have provided when Contacting the tribunal about something and make any changes (if required) before submission to complete the request.

NB Upon submission the system should activate the 'Your requests and applications' link on the case overview/hub page, and update the link status to ‘In progress’

User Story

As a Claimant, I want to be able to review the answers/comments I have entered to contact the tribunal about something, So that I can make any changes (if required) before submitting my request

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/auth/index.ts [hunk 1]
```diff
@@ -26,8 +26,9 @@ export const getUserDetails = async (
   const secret: string = config.get('services.idam.clientSecret');
   const tokenUrl: string = config.get('services.idam.tokenURL');
   const callbackUrl = encodeURI(serviceUrl + callbackUrlPageLink);
-  const code = encodeURIComponent(rawCode);
-  const data = `client_id=${id}&client_secret=${secret}&grant_type=authorization_code&redirect_uri=${callbackUrl}&code=${code}`;
+  // const code = encodeURIComponent(rawCode);
```
Comment (Itay-Kirsh) [inline src/main/auth/index.ts:29]: Did you mean to commit this? I thought this is just a local change.

File: src/main/modules/oidc/index.ts [hunk 2]
```diff
@@ -73,7 +73,7 @@ export const idamCallbackHandler = async (
 
   //For now if user account does not have the citizen role redirect to login
   if (!req.session.user?.isCitizen) {
-    return res.redirect(AuthUrls.LOGIN);
+    // return res.redirect(AuthUrls.LOGIN);
```
Comment (Itay-Kirsh) [inline src/main/modules/oidc/index.ts:76]: Another local change, don't commit

**Review by lvo700 — DISMISSED** [review]

File: src/main/auth/index.ts (hunk 1 above)
Comment (ikirsanov) [reply src/main/auth/index.ts]: Fixed that

File: src/main/modules/oidc/index.ts (hunk 2 above)
Comment (ikirsanov) [reply src/main/modules/oidc/index.ts]: Fixed that

**Review by Itay-Kirsh — APPROVED** [review]

**Review by lvo700 — APPROVED** [review]

**Pull request closed without merging**